Server/src/Config/Config.java

```diff
@@ -65,4 +69,39 @@ public String getProperty(String property) {
         return config.getProperty(property);
     }
 
+    public String getPathToCustomCards(){
+        Path path = Path.of(configFullPath.getParent().toString() + "/Custom_Cards");
+        if( Files.notExists(path) ){
+            createCustomCardsDirectory();
```

Comment:
A function called "getX" should not create something as a side-effect.  This sort of stuff is asking for bugs (for a better understanding, read up on functional programming (specifically the concept of side-effects)).  

If you need to create this directory, I'd argue you should do this at a higher level of abstraction (e.g. where this function is called)